internal/task/task/common/export_toolsv2_conf.go

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2023-11-12
+ * Author: Jiang Jun (youarefree123)
+ */
+
+package common
+
+import (
+	"fmt"
+
+	"github.com/opencurve/curveadm/cli/cli"
+	comm "github.com/opencurve/curveadm/internal/common"
+	"github.com/opencurve/curveadm/internal/configure/topology"
+	"github.com/opencurve/curveadm/internal/task/step"
+	"github.com/opencurve/curveadm/internal/task/task"
+)
+
+func NewExportToolsV2ConfTask(curveadm *cli.CurveAdm, dc *topology.DeployConfig) (*task.Task, error) {
+	serviceId := curveadm.GetServiceId(dc.GetId())
+	containerId, err := curveadm.Storage().GetContainerId(serviceId)
+	if curveadm.IsSkip(dc) {
+		return nil, nil
+	} else if err != nil {
+		return nil, err
+	} else if len(containerId) == 0 {
+		return nil, nil
+	} else if containerId == comm.CLEANED_CONTAINER_ID {
+		return nil, nil
+	}
```

Comment:
What does mean?

Reply:
I referred to the implementation of [NewCollectService](https://github.com/opencurve/curveadm/blob/580b26dd06144f327a9afe5e103bd18ff7a8a1a9/internal/task/task/common/collect_service.go#L79-L87) at that time, may be I need to evaluate whether it is necessary.